libs/application/templates/parental-leave/src/fields/Duration/index.tsx

```diff
@@ -18,21 +17,22 @@ import { FieldDescription } from '@island.is/shared/form-fields'
 
 import Slider from '../components/Slider'
 import * as styles from './Duration.treat'
-import {
-  getExpectedDateOfBirth,
-  calculatePeriodPercentageBetweenDates,
-} from '../../lib/parentalLeaveUtils'
+import { getExpectedDateOfBirth } from '../../lib/parentalLeaveUtils'
 import { parentalLeaveFormMessages } from '../../lib/messages'
 import { usageMaxMonths, usageMinMonths } from '../../config'
 import { StartDateOptions } from '../../constants'
 import { monthsToDays } from '../../lib/directorateOfLabour.utils'
 import { useGetOrRequestEndDates } from '../../hooks/useGetOrRequestEndDates'
 
 const df = 'yyyy-MM-dd'
-
 const DEFAULT_PERIOD_LENGTH = 1
+const DEFAULT_PERIOD_PERCENTAGE = 100
```

Comment:
I know, it might be not perfect, if someone is creating an application and have less than 1 month of rights. We could move that into a VMST API madness hooks to get startDate/endDate/percentage from multiple calls